README.md

```diff
@@ -18,7 +18,7 @@ torch = PyTorchUtils.PyTorchUtilsLogic().torch  # will be installed if necessary
 tensor = torch.rand(50, 60)
 ```
 
-Please use a Preview Release of 3D Slicer (currently 4.13, **not** 4.11).
+Please use a **Preview** Release of 3D Slicer.
```

Comment:
PyTorch is available on the current Slicer Stable Release, so this comment is no longer needed.
```suggestion
```